Tools/CISupport/ews-build/steps.py

```diff
@@ -6564,9 +6564,16 @@ def run(self, BufferLogObserverClass=logobserver.BufferLogObserver):
             defer.returnValue(rc)
             return
 
-        self.contributors, errors = yield Contributors.load(use_network=True)
-        for error in errors:
-            self._addToLog('stdio', error)
+        retry_attempts = 3
+        for i in range(1, retry_attempts + 1):
+            self.contributors, errors = yield Contributors.load(use_network=True)
+            if self.contributors:
+                break
+            for error in errors:
```

Comment:
we can move this above, before the break. In case there are errors as well as self.contributors: